doc/lightning-listrevokeinfos.7

```diff
@@ -0,0 +1,152 @@
+'\" t
+.\"     Title: lightning-listrevokeinfos
+.\"    Author: [see the "AUTHOR" section]
+.\" Generator: DocBook XSL Stylesheets v1.79.1 <http://docbook.sf.net/>
+.\"      Date: 04/15/2018
+.\"    Manual: \ \&
+.\"    Source: \ \&
+.\"  Language: English
+.\"
+.TH "LIGHTNING\-LISTREVOK" "7" "04/15/2018" "\ \&" "\ \&"
+.\" -----------------------------------------------------------------
+.\" * Define some portability stuff
+.\" -----------------------------------------------------------------
+.\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.\" http://bugs.debian.org/507673
+.\" http://lists.gnu.org/archive/html/groff/2009-02/msg00013.html
+.\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.ie \n(.g .ds Aq \(aq
+.el       .ds Aq '
+.\" -----------------------------------------------------------------
+.\" * set default formatting
+.\" -----------------------------------------------------------------
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.\" -----------------------------------------------------------------
+.\" * MAIN CONTENT STARTS HERE *
+.\" -----------------------------------------------------------------
+.SH "NAME"
+lightning-listrevokeinfos \- List revocation info for channels, for giving to *TRUSTED* WatchTowers\&.
+.SH "SYNOPSIS"
+.sp
+\fBlistrevokeinfos\fR [\fIchannel_id\fR]
+.SH "DESCRIPTION"
+.sp
+The \fBlistrevokeinfos\fR RPC command returns revocation information for all open channels, or for a specific channel\&. The intent is to somehow pass this revocation information to some \fBTRUSTED\fR WatchTower\&.
+.sp
+Note that this requires a \fBTRUSTED\fR WatchTower: it contains information that would let your couterparty steal funds outright while you are offline\&. You are trusting the WatchTower not to leak this information to channel counterparties\&.
```

Comment:
> it contains information that would let your couterparty steal funds outright while you are offline

How is it different from not having watch tower? Without watch tower a counter party can do the same thing.

Also typo *couterparty*

Reply:
Without this, the counterparty has to wait out the entire timelock before it can steal the funds.  If you manage to come online during the timelock, you can revoke the stolen funds.

With cooperation of a WatchTower that has access to `our_revocation_basepoint_secret`, the counterparty does not have to wait at all: it can steal the funds immediately.  Basically: counterparty publishes revoked transaction, WatchTower revokes it *to the counterparty and WatchTower splitting the profits 50/50*.  This is strictly worse and requires that the WatchTower be trusted.  You also cannot safely form a direct channel with the WatchTower.